### PR TITLE
Fix tests with timeout.

### PR DESCRIPTION
**Pull request by dzeckelev**
Resolves BV-829.

### CODE REVIEW

File: client/bill/monitor_test.go [hunk 1]
```diff
@@ -80,6 +66,47 @@ func newFixture(t *testing.T, db *reform.DB) *data.TestFixture {
...
+}
+
+func awaitingGoodPosting(wg *sync.WaitGroup, postErrors chan error) {
+	defer wg.Done()
+	for {
+		select {
+		case e := <-postErrors:
+			if e == nil {
+				return
+			}
```
Comment (furkhat) [inline client/bill/monitor_test.go]: is this check redundant?

Comment (dzeckelev) [reply client/bill/monitor_test.go]: no. If the error has become is nil, then you need to complete the function.

Comment (furkhat) [reply client/bill/monitor_test.go]: ok, but it will exit anyway.

Comment (dzeckelev) [reply client/bill/monitor_test.go]: Here an infinite loop. The loop will end either by timeout, or if a error is nul

**Pull request merged**